src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -486,30 +479,53 @@ protected void verify(Checker checker,
      */
     protected final void verify(Checker checker,
                           File[] processedFiles,
-                          Map<String, List<String>> expectedViolations)
-            throws Exception {
+                          Map<String, List<String>> expectedViolations) throws Exception {
         stream.flush();
         stream.reset();
-        final List<File> theFiles = new ArrayList<>();
-        Collections.addAll(theFiles, processedFiles);
-        final int errs = checker.process(theFiles);
```

Comment:
should consider supplemental PR migrating the flow premature, to only having this conversion to `Path` in diff.